DevDashboard/cloud/landing/field-notes/index.html

```
@@ -0,0 +1,716 @@
+            <div class="flex items-center justify-between">
+              <div class="flex items-center gap-2">
+                <span class="grid h-6 w-6 place-items-center rounded-full bg-oxblood/30 text-[10px] font-semibold text-cream">CC</span>
+                <span class="text-[12px] font-medium text-cream/90">Claude Code · refactor/auth</span>
+              </div>
+              <span class="rounded-full bg-oxblood/25 px-2 py-0.5 text-[9px] uppercase tracking-[0.18em] text-oxblood-soft">needs you</span>
+            </div>
+            <p class="mt-2.5 text-[11px] leading-relaxed text-cream/55">
+              <span class="text-cream/80">Agent paused —</span> apply migration to <span class="font-mono text-cream/80">prod.db</span>? Awaiting approval.
+            </p>
+            <div class="mt-3 flex gap-2">
+              <button class="flex-1 rounded-lg bg-oxblood py-1.5 text-[11px] font-medium text-cream">Approve</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)